romancal/resample/resample_utils.py

```diff
@@ -102,3 +102,26 @@ def make_output_wcs(
     )
 
     return wcs, pscale * 3600.0, pscale_ratio
+
+
+def compute_var_sky(model) -> None:
+    """
+    Add sky variance array to a datamodel.
+
+    Parameters
+    ----------
+    model : `ImageModel`
+        A datamodel to which the sky variance array will be added.
+
+    Returns
+    -------
+    None
+    """
+
+    ok_data = model["data"] != 0
+    var_sky = model["var_rnoise"].copy()
+    var_sky[ok_data] = model["var_rnoise"][ok_data] + model["var_poisson"][
+        ok_data
+    ] / model["data"][ok_data] * np.median(model["data"])
```

Comment:
```suggestion
    ] / model["data"][ok_data] * np.median(model["data"])
```
Is there any case in which `model["data"]` will contain a `nan`?